src/index.js

```diff
@@ -32,14 +40,15 @@ function setupLocalDataTrack() {
 
   window.addEventListener('mousemove', event => {
     const { pageX: x, pageY: y } = event;
-    mouseCoordinates = { x, y };
+    mouseCoordinates = { x: x / window.innerWidth, y: y / window.innerHeight };
 
     if (mouseDown) {
       const color = colorHash.hex(dataTrack.id);
       drawCircle(canvas, color, x, y);
 
+      // Send the drawing coordinate
       dataTrack.send(JSON.stringify({
-        mouseDown,
+        mouseDown: false,
```

Comment:
@aaalaniz If we send the proper value of `mouseDown` here, I think we can add back the `if (mouseDown)` block that you've removed below, right? Or since we're sending coordinates only when `mouseDown === true`, then we could even remove `mouseDown` from the payload.

WDYT @markandrus ?